controllers/release/controller.go

```diff
@@ -82,8 +82,9 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		adapter.EnsureFinalizerIsAdded,
 		adapter.EnsureReleaseExpirationTimeIsAdded,
 		adapter.EnsureTenantPipelineIsProcessed,
+		adapter.EnsureReleaseTenantProcessingIsTracked,
```

Comment:
I'd rename these functions to `EnsureTenantPipelineProcessingIsTracked` and `EnsureManagedProcessingIsTracked`.

Reply:
done